cinema/serializers.py

```diff
@@ -68,13 +80,79 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
 class MovieSessionDetailSerializer(MovieSessionSerializer):
     movie = MovieListSerializer(many=False, read_only=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
+    taken_places = SerializerMethodField()
```

Comment:
You can use nested serializer here - create TicketSeatsSerializer with row / seat fields and pass it to taken_places